src/app/garfile/responsibleperson/index.njk

```diff
@@ -58,6 +58,20 @@
               aria-describedby="responsibleContactNo-hint{{ ' responsibleContactNo-error' if errors | containsError('responsibleContactNo') }}">
           </div>
 
+          <div class="govuk-form-group govuk-!-margin-bottom-6">
```

Comment:
Have you tested the error validation for this field? Were the errors displayed properly? Can you please check that as one of the macros to add a css class for error seems to be missing here. It is present in a few other fields in this file.  
`{{ m.form_group_class('responsibleContactNo', errors) }}`

Reply:
Absolutely, I should have provided the error messages as part of the screenshots.

**Below is the error messages for:**

The top page if the email address is missing 
<img width="1439" alt="Screenshot 2023-04-14 at 14 03 46" src="https://user-images.githubusercontent.com/123387957/232055619-41b1670c-379c-49bf-a151-59f7346aead9.png">

The error nearer the input field.
<img width="1439" alt="Screenshot 2023-04-14 at 14 03 54" src="https://user-images.githubusercontent.com/123387957/232055636-151bb144-1ea1-4433-9ba1-8ad98ca91419.png">